api/services/users/userServices.js

```diff
@@ -30,16 +30,10 @@ const authenticate = async ({ email, password }) => {
   const [user] = await collection
     .aggregate(userPipelines.authenticate(email))
     .toArray();
-  if (!user.emailVerified) throw "Email is not verified";
   if (user && bcrypt.compareSync(password, user.hash)) {
-    const { hash, emailVerified, ...userWithoutHash } = user;
-
+    const { hash, ...userWithoutHash } = user;
```

Comment:
check this logic

Reply:
I didn't find a way to improve this logic